runtime/redirector/CMakeLists.txt

```diff
@@ -56,10 +56,18 @@ if(OMR_OS_ZOS)
 	# Ensure that omrutil is built before we attempt to link it.
 	add_dependencies(jvm_redirect omrutil)
 endif()
+
+if(OMR_OS_ZOS)
+	set(archive_dir ${j9vm_BINARY_DIR}/lib/redirector)
```

Comment:
How come z/OS is different from other platforms?
and why do all the platforms need to set ARCHIVE_OUTPUT_DIRECTORY when it wasn't set before?

Reply:
>How come z/OS is different from other platforms?

I'm not sure why we make the distinction, but the jdk extensions look in a different location on Windows vs z/OS (with Windows being the only other platform where we care about the import libraries. eg the .lib and .x files that accompany shared libs)
see
https://github.com/ibmruntimes/openj9-openjdk-jdk11/blob/376b14b921f110c22454b1c12b348ed1856398aa/closed/custom/copy/Copy-java.base.gmk#L89-L112

>why do all the platforms need to set ARCHIVE_OUTPUT_DIRECTORY when it wasn't set before?

This is where UMA currently generates its static libraries. Currently the only place it actually matters is for the redirector and jsig on windows, however on z/OS the jdk extensions link against a number of different libraries and expect the `.x` files to live in `/lib`

Reply:
I see, Windows puts files in the bin directory. I think it would make more sense to check for Windows to use `${CMAKE_CURRENT_BINARY_DIR}`, and then make `${j9vm_BINARY_DIR}/lib/redirector` the else case.

Reply:
To put it another way, it's Windows which is different from other platforms because it puts things in bin. I expect that any other platform that uses .lib files should have the same layout as z/OS.